concepts/05 Widgets/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,212 @@
+You can modify any form, editor or item properties at runtime. In **jQuery**, use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Widgets/dxForm/Methods/#optionoptionName_optionValue) method to update a property value. In **Angular**, **Vue** or **React**, bind the property's value to a component property.
```

Comment:
>You can modify any form, editor or item properties at runtime.

"any form"? May be it should be "any form option"?

"editor or item properties" - how to implement these scenarios?

Reply:
This should read "any form property, editor property or item property". However, I think that I'll try to rephrase this sentence to make it more clear.

>doesn't provide links to instructions
Good point. I'll think about how to do it.

Reply:
I meant any properties of form, editor or item. In one of the previous snippets we've seen how to customize an editor.

But I'll try to make it more clear.